### PR TITLE
Select or Create in Menu

### PR DESCRIPTION
**Pull request by brianteeman**
### Summary of Changes
We now have the great ability to create an article, contact, category directly at the point you create a menu item. This PR updates the tooltip displayed to reflect that you can Select AND create. I also took the opportunity to standardise the strings across the component for consistency

### Testing Instructions
On review or by checking the tooltips when creating menu items
screenshotr12-23-52

### CODE REVIEW

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 43cf3cb3880ecd27c5dac9d8c170018406170eb5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14637">issues.joomla.org/tracker/joomla-cms/14637</a>.</sub>

**Comment by jeckodevelopment** [comment]
I have tested this item :white_check_mark: successfully on 43cf3cb3880ecd27c5dac9d8c170018406170eb5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14637">issues.joomla.org/tracker/joomla-cms/14637</a>.</sub>

**Comment by jeckodevelopment** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14637">issues.joomla.org/tracker/joomla-cms/14637</a>.</sub>

File: administrator/language/en-GB/en-GB.ini [hunk 1]
```diff
@@ -305,7 +305,7 @@ JGLOBAL_CATEGORY_ORDER_DESC="The order that categories will show in."
 JGLOBAL_CATEGORY_ORDER_LABEL="Category Order"
 JGLOBAL_CENTER="Center"
 JGLOBAL_CHECK_ALL="Check All"
-JGLOBAL_CHOOSE_CATEGORY_DESC="Choose a category from the list."
+JGLOBAL_CHOOSE_CATEGORY_DESC="Select  or create a category to be displayed."
```
Comment (wilsonge) [inline administrator/language/en-GB/en-GB.ini]: OK Two things here:
1. You have a double space beween "select" and "or"
2. Given this is in the global file and people may not have the create category option enabled do we feel comfortable that this isn't against b/c?

**Review by wilsonge — CHANGES_REQUESTED** [review]

**Comment by franz-wohlkoenig** [comment]
this PR have no Milestone.

**Pull request merged**